library/core/class.vanillahtmlawed.php

```diff
@@ -1,18 +1,13 @@
 <?php
 /**
- * HtmLawed Plugin.
- *
  * @copyright 2009-2017 Vanilla Forums Inc.
  * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
- * @package HtmLawed
  */
 
-use Garden\Container\Container;
-
 /**
- * Class HTMLawedPlugin
+ * Class VanillaHtmlFormatter
  */
-class HtmLawedPlugin extends Gdn_Plugin {
+class VanillaHtmlFormatter {
```

Comment:
This class is in a file named "class.vanillahtmlawed.php". The filename should be updated to reflect the class name.